test/management_tests/procedures_test.py

```diff
@@ -594,3 +594,152 @@ async def test_nm_individual_address_write_address_found_other_in_programming_mo
 
     with pytest.raises(ManagementConnectionError):
         await task
+
+
+async def test_nm_individual_address_serial_number_read(time_travel):
+    """Test nm_individual_address_serial_number_read."""
+
+    xknx = XKNX()
+    xknx.cemi_handler = AsyncMock()
+    individual_address = IndividualAddress("1.1.5")
+    serial_number = b"aabbccddeeff"
+
+    task = asyncio.create_task(
+        procedures.nm_individual_address_serial_number_read(
+            xknx=xknx, serial=serial_number
+        )
+    )
+
+    read_address = Telegram(
+        destination_address=GroupAddress("0/0/0"),
+        payload=apci.IndividualAddressSerialRead(serial=serial_number),
+    )
+    address_reply = Telegram(
+        source_address=individual_address,
+        destination_address=GroupAddress("0/0/0"),
+        direction=TelegramDirection.INCOMING,
+        payload=apci.IndividualAddressSerialResponse(
+            address=individual_address, serial=serial_number
+        ),
+    )
+
+    await time_travel(0)
+    xknx.management.process(address_reply)
+    await time_travel(0.5 * management.management.MANAGAMENT_CONNECTION_TIMEOUT)
+    assert xknx.cemi_handler.send_telegram.call_args_list == [
+        call(read_address),
+    ]
```

Comment:
```suggestion
    await time_travel(0)
    assert xknx.cemi_handler.send_telegram.call_args_list == [
        call(read_address),
    ]
    xknx.management.process(address_reply)
```
we don't need to await the timeout when the reply was received.